<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T_Corp POS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="ShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame 
{
	color: rgb(199, 199, 199);
	background-color: rgb(9, 9, 9);
	border-radius: 10
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>571</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gabriola</family>
         <pointsize>45</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(7, 202, 147);</string>
       </property>
       <property name="text">
        <string>T_Corp&lt;strong&gt; POS&lt;/strong&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(159, 159, 159);</string>
       </property>
       <property name="text">
        <string>Powered by Mark_T</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>551</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar
{
	background-color: rgb(11, 11, 11);
	color: rgb(26, 240, 158);
	border-radius: 10px;
	text-align: center;
}
	QProgressBar::chunk{
background-color: qlineargradient(spread:pad, x1:0.011, y1:0.512, x2:1, y2:0.54, stop:0.181818 rgba(5, 48, 43, 255), stop:1 rgba(0, 255, 255, 255));
border-radius: 10;
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>581</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(159, 159, 159);</string>
       </property>
       <property name="text">
        <string>Loading Files...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_version">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>360</y>
         <width>100</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>11</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(159, 159, 159);</string>
       </property>
       <property name="text">
        <string>v 1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
